agent/local/state.go

```diff
@@ -1082,27 +1082,29 @@ func (l *State) updateSyncState() error {
 			continue
 		}
 
+		svc := *ls.Service
```

Comment:
Might be worth explaining why this is done so someone doesn't come by and 'optimize' it out.